src/test/resources/org/broadinstitute/hellbender/utils/interval/bad_interval_span.bed

```diff
@@ -0,0 +1 @@
+1	249240000	249260000
```

Comment:
The name of this bed file should indicate which reference it uses (eg., `bad_hg19_interval.bed`)